ai_chatbots/chatbots.py

```diff
@@ -546,13 +546,7 @@ async def get_completion(
 
             await create_tutorbot_output(self.thread_id, json_output)
             prossessed = process_StratL_json_output(json_output)
-            response = "An error has occurred, please try again"
```

Comment:
Since the output is now a normal Langraph output and not a tool call, this is no longer needed